dev/integration_tests/ios_app_with_extensions/ios/Runner.xcodeproj/project.pbxproj

```diff
@@ -630,7 +629,7 @@
 				MTL_ENABLE_DEBUG_INFO = INCLUDE_SOURCE;
 				MTL_FAST_MATH = YES;
 				OTHER_LDFLAGS = "";
-				PRODUCT_BUNDLE_IDENTIFIER = com.example.watchCompanionTest.watchkitapp;
+				PRODUCT_BUNDLE_IDENTIFIER = io.flutter.extensionTest.watchkitapp;
```

Comment:
this should probably be `dev.flutter.extensionTest.watchkitapp`, but that's probably a bigger refactor for another day